### DISCUSSION TITLE
Is Svelte a requrement?

### DISCUSSION THREAD

**Discussion by jnegron357** [vkurko/calendar#93] [Q&A] [answered]
The Read me does not mention or give an example of using this with a regular HTML page, is this possible? To clarify, I want to use the downloaded source to load the library from a local directory. The browser example only shows the CDN option. Which files do I load locally and in what order?

**Comment by vkurko** [comment, marked as answer]
No, Svelte is not a requirement. You can use the library in any JS application.

A browser version is available in the [@event-calendar/build](https://www.npmjs.com/package/@event-calendar/build) npm package. The CDN in fact takes the files from this package. Therefore, you can simply download the necessary js and css files from these links:

https://cdn.jsdelivr.net/npm/@event-calendar/build/event-calendar.min.js
https://cdn.jsdelivr.net/npm/@event-calendar/build/event-calendar.min.css

and use them as follows (let's assume you put the files in the `assets` folder at the root of your webserver):

```html
<link rel="stylesheet" href="/assets/event-calendar.min.css">
<script src="/assets/event-calendar.min.js"></script>
```

**Reply by jnegron357** [reply]
Thank you for your reply, I have a follow up question: Is there a way to get the unminified version of those files from the CDN?

**Reply by vkurko** [reply]
> Thank you for your reply, I have a follow up question: Is there a way to get the unminified version of those files from the CDN?

The library provides [source map](https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html) file, so for example in the debugger you should see unminified code. There are some [tools](http://sokra.github.io/source-map-visualization) for visualizing the source map, and you can also [compile](https://github.com/vkurko/calendar/discussions/59) the library yourself with the parameters you need.
